doc/sal_constraints_and_recommendations.rst

```diff
@@ -452,6 +453,14 @@ e.g. for ATTCS in C++:
     ATTCS_shared_SimpleSetA (declared globally)
     ATTCS_someEvent_SpecificSetA (datum specific)
 
+Define it using the following format.
+
+1. In ``CSCName_Events.xml`` create an ``Enumeration`` tag just before the first ``SAlEvent`` tag in the file.
+2. Fill out the Enumeration with comma delimited values using the following naming format ``itemName_value``.
+3. Repeat for as many enumerations as you require.
+
+.. warning:: Some enumerations are defined inside of the topic themselves. This is a legacy method and no longer recommended or necessary.
+
```

Comment:
Shouldn't you mention you shall (optionally) add enumerations values as constant to ts_idl/python/lsst/ts/idl/enums/<CSCName.py? At least before that would be auto-generated?